packages/common/src/index.ts

```diff
@@ -538,17 +538,15 @@ export default class Common extends EventEmitter {
    *
    * @param topic Parameter topic ('gasConfig', 'gasPrices', 'vm', 'pow')
    * @param name Parameter name (e.g. 'minGasLimit' for 'gasConfig' topic)
-   * @returns The value requested or `null` if not found
+   * @returns The value requested or `undefined` if not found
    */
-  param(topic: string, name: string): any {
+  param(topic: string, name: string): bigint {
     // TODO: consider the case that different active EIPs
     // can change the same parameter
-    let value = null
+    let value
     for (const eip of this._eips) {
       value = this.paramByEIP(topic, name, eip)
-      if (value !== null) {
-        return value
-      }
+      if (value !== BigInt(0)) return value
```

Comment:
Oops

Reply:
Ah!  I figured it it was something like that.  Thanks!

Reply:
How do we solve this? We could hardcode a -1 bigint but I don't think that's clean, shall we fix the function signatures such that these can also return `null`?

Reply:
Wait..no, that shouldn't be it.  paramByEIP will always return a bigint now (either 0n or the actual value).